src/components/Overlay/Readme.mdx

```diff
@@ -0,0 +1,87 @@
+---
+name: Overlay
+route: /components/Overlay
+menu: Components
+---
+
+import { Props, Playground } from 'docz';
+import { FiMenu, FiX, FiChevronUp } from 'react-icons/fi';
+import { Flex } from '../Flex';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { 
+  Expandable, 
+  ExpandableToggle, 
+  ExpandableBody,
+  // ExpandableContext 
+} from '../Expandable';
+import { HeartLoader } from 'components/Loader';
```

Comment:
for docz 2.2:
`import { HeartLoader } from '../Loader';`

Reply:
think that's the only realy change you need to make it work with master/docz2.2

then just run `yarn run build && yarn run prettier:all` again and the rest of the conflicts should be resolved.